# simple message for object volume and density

# weight
float32 convex_area
float32 convex_volume
float32 convex_density

float32 height
float32 width
float32 depth

float32 mean_0
float32 mean_1
float32 mean_2
float32 mean_3

# time estimation
time time_now